ESP_ATMod/command.cpp

```diff
@@ -121,6 +122,17 @@ bool readNumber(unsigned char *inpBuf, uint16_t &offset, uint32_t &output);
 bool readIpAddress(unsigned char *inpBuf, uint16_t &offset, uint32_t &output);
 uint8_t readHex(char c);
 void printCertificateName(uint8_t certNumber);
+int compWifiRssi(const void *elem1, const void *elem2);
+void printCWLAP(int networksFound);
+void prinScanResult(int networksFound);
+
+/*
+ * Variables
+ */
+uint32_t sort_enable = 0;
+uint32_t printMask = 0x7FF;
+uint32_t rssiFilter = -100;

```

Comment:
When initializing with a negative value, signed type should be used: `int`

Reply:
Resolved.